src/app/AppLayout/AppLayout.tsx

```diff
@@ -409,9 +407,7 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
                     data-quickstart-id="settings-link"
                     component={(props) => <Link {...props} to="/settings" />}
                   >
-                    <Icon size="sm">
-                      <CogIcon />
-                    </Icon>
+                    <CogIcon />
```

Comment:
We might still need to wrap `<CogIcon />` in `Icon` component?

```jsx
<Icon>
  <CogIcon />
</Icon>
```

There are some wrapper elements when inspecting the DOM but mostly because the docs here use it everywhere: https://www.patternfly.org/components/icon.